src/Ledger/TokenAlgebra.lagda

```diff
@@ -1,40 +1,53 @@
 \section{Token algebras}
+\label{sec:token-algebra}
+
 \begin{code}[hide]
 {-# OPTIONS --safe #-}
-module Ledger.TokenAlgebra where
+
+open import Agda.Primitive  using() renaming (Set to Type)
+
+module Ledger.TokenAlgebra (PolicyId : Type) where
 
 open import Ledger.Prelude
 
-open import Algebra using (CommutativeMonoid)
-open import Algebra.Morphism
-open import Data.Integer
-open import Data.Nat.Properties using (+-0-commutativeMonoid)
+open import Agda.Primitive       using() renaming (Set to Type)
+open import Algebra              using (CommutativeMonoid ; Monoid)
+open import Algebra.Morphism     using (module MonoidMorphisms )
+open import Data.Nat.Properties  using (+-0-monoid)
+open import Relation.Binary      using (Rel)
+open import Relation.Unary       using (Pred)
 \end{code}
 
 \begin{figure*}
 \begin{code}
 record TokenAlgebra : Set₁ where
-  field PolicyId : Set
-        Value-CommutativeMonoid : CommutativeMonoid 0ℓ 0ℓ
+  field  Value-CommutativeMonoid : CommutativeMonoid 0ℓ 0ℓ
 
+  MemoryEstimate : Type
   MemoryEstimate = ℕ
 \end{code}
 \begin{code}[hide]
-  open CommutativeMonoid Value-CommutativeMonoid using (_≈_; ε)
-       renaming (Carrier to Value; refl to reflᵛ; _∙_ to _+ᵛ_) public
+  open CommutativeMonoid Value-CommutativeMonoid public
+    using ( _≈_ ; ε ; monoid ; rawMonoid)
```

Comment:
Is there a reason for changing it like that? I find this more difficult to read

Reply:
I don't love it either, but it's recommended in the standard library style guide.  I guess they want it to be very easy to spot the `public` directive.

I've been changing most public imports to appear like that, for consistency and to follow the standard library recommendation... but let me know if you hate it and I'll change this and all other such occurrences back.

Reply:
My main complaint is that the line breaks seem arbitrary. If you want to move `public` to the front, that's fine with me, but there's no reason to split this into 6 lines from the original 2.